r/R/dplyr-mutate.R

```diff
@@ -151,3 +153,95 @@ ensure_named_exprs <- function(exprs) {
   names(exprs)[unnamed] <- map_chr(exprs[unnamed], format_expr)
   exprs
 }
+
+# Take the input quos and unfold any instances of across()
+# into individual quosures
+unfold_across <- function(.data, quos_in) {
```

Comment:
How does this compare/contrast to how dplyr does this?

Reply:
> How does this compare/contrast to how dplyr does this?

For clarity, why do you ask? 

Or is this less of a question, and actually you're just tactfully saying "hey Nic, might be worth looking through `dplyr::across()` as you could just do it how it's done there instead of coming up with it all over again with bits missing"?  (If so, thanks for pointing this out, as I keep forgetting this!)

Reply:
OK, have now updated this using `dplyr::across()` as reference.

Reply:
(FWIW it was already pretty similar by chance, but I used a lot of the code there to update things like weird edge cases and how the naming stuff works)